s28838-OskarSzyszko/Pamietnik.txt

```diff
@@ -0,0 +1,9 @@
+2024-03-20
+Moj kochany pamietniczku na dzisiejszych zajeciach :
+-powtozylem jak sie twozy wirtualne maszyny, czego sie nauczylem uczeszczajac do technikum
+
+Prowadzoncy przyblizył nam plan naucznia na ten semestr, a nastepnie opisal instalacje Debian'a w VirtualBox.
+
+Zaskoczyla mnie troche ilosc wszystkich opcji przy instalacji Debian'a ponieraz w technikum nie zaglembialismy sie az tak w ten temat.
```

Comment:
Tam było technikum. Tu są studia. Po studiach dostajesz tytuł inżyniera a to coś jednak oznacza.